dotcom-rendering/src/components/VideoYoutubeBlockComponent.amp.tsx

```diff
@@ -10,7 +10,7 @@ type Props = {
 export const VideoYoutubeBlockComponent = ({ element, pillar }: Props) => {
 	const youtubeId = getIdFromUrl(
 		element.originalUrl || element.url,
-		'^[a-zA-Z0-9_-]{11,}$', // Alphanumeric, underscores and hyphens, 11 or more characters long
+		'^[a-zA-Z0-9_-]{11}$', // Alphanumeric, underscores and hyphens, 11 characters long
```

Comment:
In lieu of a test for `VideoYoutubeBlockComponent.amp.tsx` could this regex be exported and used in `get-video-id.amp.test.ts` ? I think this would ensure the test is exercising the regex used by the implementation. Looks like they have gone out of sync slightly?

https://github.com/guardian/dotcom-rendering/blob/814ab3fd2d520a86915c1ab8a1acbe1d9edc2109/dotcom-rendering/src/lib/get-video-id.amp.test.ts#L5

Reply:
uh oh, good spot thank you